### PR TITLE
[GH-287] Move cardano-shell into subpackage.

### PR DESCRIPTION
**Pull request by ksaric**
https://github.com/input-output-hk/cardano-shell/issues/287

### CODE REVIEW

**Comment by ksaric** [comment]
A similar error I encountered locally:

**Comment by ksaric** [comment]
Coveralls seems to be working :tada:

**Comment by ksaric** [comment]
Coveralls working :tada: 

https://github.com/input-output-hk/cardano-shell/issues/280

**Comment by ksaric** [comment]
One thing left to fix in a separate issue is the failing test:
```
    DaedalusIPC
      DaedalusIPC
        should reply with the port when asked FAILED [1]
 
    Failures:
 
      test/DaedalusIPCSpec.hs:17:9:
      1) DaedalusIPC.DaedalusIPC should reply with the port when asked
           expected: ExitSuccess
            but got: ExitFailure 1
 
      To rerun use: --match "/DaedalusIPC/DaedalusIPC/should reply with the port when asked/"
 
    Randomized with seed 1490635108
 
    Finished in 4.5110 seconds
    14 examples, 1 failure
```

File: specs/iohk.sty [hunk 1]
```diff
@@ -1,43 +0,0 @@
...
-  {\begin{bclogo}[logo=\bcquestion, couleur=orange!10, arrondi=0.2]{ QUESTION}}
-  {\end{bclogo}}
-\newenvironment{todo}
-  {\begin{bclogo}[logo=\bcoutil, couleur=red!5, couleurBarre=red, arrondi=0.2]{ TODO}}
-    {\end{bclogo}}
-\newenvironment{note}
-  {\begin{bclogo}[logo=\bcattention, couleur=orange!10, arrondi=0.2]{ NOTE}}
-  {\end{bclogo}}
-
-\endinput
```
Comment (HirotoShioi) [inline specs/iohk.sty]: Can you recover this file? We need this to compile Latex file. I get this error when trying to compile it.

```
[nix-shell:~/haskell/cardano-shell/cardano-shell/specs]$ make all
latexmk -pdf -pdflatex="pdflatex -interaction=nonstopmode" -use-make CardanoShellSpec.tex
Latexmk: This is Latexmk, John Collins, 7 August 2018, version: 4.59.
Latexmk: 'pdflatex': source file 'iohk.sty' doesn't exist. I'll try making it...
------------
Running 'make "iohk.sty"'
------------
make[1]: Entering directory '/home/hiroto/haskell/cardano-shell/cardano-shell/specs'
make[1]: *** No rule to make target 'iohk.sty'.  Stop.
make[1]: Leaving directory '/home/hiroto/haskell/cardano-shell/cardano-shell/specs'
Latexmk: All targets (CardanoShellSpec.pdf) are up-to-date
```

Comment (ksaric) [reply specs/iohk.sty]: Good catch.

Comment (ksaric) [reply specs/iohk.sty]: BTW, you need to first drop into the `nix-shell` in order to run Make on the specs.

Comment (HirotoShioi) [reply specs/iohk.sty]: Yes that's what I did and this is the error message I got.
```
[nix-shell:~/haskell/cardano-shell/cardano-shell/specs]$ make all
```

**Pull request merged**